### ISSUE TITLE
Denite seems to be broken after updating to latest master

### ISSUE THREAD

**Issue by ccojocar** [wsdjeg/SpaceVim#2881]
The fallowing error is returned by the statusline and the fuzzy search seems to be broken:

```

Error detected while processing function SpaceVim#layers#core#statusline#denite_mode:
line    2:
E117: Unknown function: denite#get_status_mode
Press ENTER or type command to continue
``` 

I tried to use `mode()` instead but it seems that also `get_status_sources` to not be available anymore:

```

E117: Unknown function: denite#get_status_sources
E15: Invalid expression: denite#get_status_sources()
```

I think the `statusline.vim` needs some update to be compatible with latest version of denite.
 
## The reproduce ways from Vim starting (Required!)

```
Spacevim version : 1.2.0-dev
NVIM v0.3.5
Build type: Release
```

## Debug info

Please press <kbd>SPC h I</kbd>, debug info will be put into clipboard, then paste all content below.

## Screenshots

If you have any screenshots for this issue, please upload here. BTW you can use https://asciinema.org/ for recording video in terminal.

**Comment by wsdjeg** [comment]
It is known bug, because only denite 2.1 is supported in spacevim now.

**Comment by ccojocar** [comment]
Is it possible to define the version in the `init.toml` e.g. in the denite layer? Thanks

**Comment by wsdjeg** [comment]
```
[options]
      disabled_plugins = ["denite.nvim"]

[[custom_plugins]]
    name = "SpaceVim/denite.nvim"
    merged = false
    loadconf = true
```

**Comment by ccojocar** [comment]
Thanks! It seems to work.

**Comment by wsdjeg** [comment]
welcome，and this bug will be fixed after #2826 is merged.

**Comment by stfl** [comment]
I had to manually delete Shougo's repo from `~/.cache/vimfiles/repos/github.com/Shougo/denite.nvim`

now the SpaceVim/denite.nvim plugin loads but the original config does not work. for example key bindings in <kbd>[SPC] p f</kbd> don't work as expected..

**Comment by wsdjeg** [comment]
merged in #3058
